build_projects/Microsoft.DotNet.Cli.Build.Framework/TargetConditions/BuildPlatformsAttribute.cs

```diff
@@ -18,11 +19,17 @@ public BuildPlatformsAttribute(params BuildPlatform[] platforms)
             _buildPlatforms = platforms;
         }
 
+        public BuildPlatformsAttribute(BuildPlatform platform, string version)
+        {
+            _buildPlatforms = new BuildPlatform[] { platform };
+            _version = version;
+        }
+
         public override bool EvaluateCondition()
         {
             foreach (var platform in _buildPlatforms)
             {
-                if (CurrentPlatform.IsPlatform(platform))
+                if (CurrentPlatform.IsPlatform(platform, _version))
```

Comment:
The way I did this, you are only allowed to specify the version when you have a single Platform.